thunder/tests/opinfos.py

```diff
@@ -542,6 +542,23 @@ def _fn(x):
     return _fn
 
 
+@wraps(torch.Tensor.is_cuda)
+def _is_cuda_torch(x: torch.Tensor):
+    # Should Number be accepted?
```

Comment:
Just accept `torch.Tensor` seems fine. 

Nit: the return value of this function could be annotated as `bool`

Reply:
This is a nice helper for the OpInfo